backend/examples/load-test-cagg.js

```
@@ -0,0 +1,45 @@
+    { duration: '30s', target: 5000 }, // Ramp up to 5k users
+    { duration: '1m', target: 5000 },  // Sustain 5k users
+    { duration: '30s', target: 10000 }, // Ramp up to 10k users
+    { duration: '1m', target: 10000 },  // Sustain 10k users
+    { duration: '30s', target: 0 },    // Ramp down to 0
+  ],
+  thresholds: {
+    http_req_duration: ['p(99)<15'], // 99% of requests must complete below 15ms
+  },
+};
+
+const BASE_URL = __ENV.API_URL || 'http://localhost:3000';
```

ESLint

[error] '__ENV' is not defined.

(no-undef)